compiler/rustc_feature/src/unstable.rs

```diff
@@ -475,6 +475,8 @@ declare_features! (
     (unstable, fn_align, "1.53.0", Some(82232)),
     /// Support delegating implementation of functions to other already implemented functions.
     (incomplete, fn_delegation, "1.76.0", Some(118212)),
+    /// Support ExitStatusExt::aborted_code on Fuchsia platform.
+    (unstable, fuchsia_exit_status_aborted, "1.79.0", None),
```

Comment:
you don't need to touch the compiler, you can just add library features

Reply:
Ah, ok. I was led astray by https://rustc-dev-guide.rust-lang.org/implementing_new_features.html. That makes sense though. I see now that this not a language feature.